frontend/src/Student/containers/ViewScholarshipsTab.js

```diff
@@ -0,0 +1,27 @@
+import { compose } from 'recompose'
+import { connect } from 'react-redux'
+import { graphql } from "react-apollo";
+
+import ScholarshipsQuery from "../queries/scholarships.graphql"
+import ViewScholarshipsTab from "../components/ViewScholarshipsTab";
+import { displayError } from "../../shared/actions";
+
+const mapDispatchToProps = dispatch => ({
+  handleError: message => dispatch(displayError(message))
+});
+
+export default compose(
+  connect(null, mapDispatchToProps),
+  graphql(ScholarshipsQuery, {
```

Comment:
the results of the scholarship query are passed to the ViewScholarshipsTab component as properties